sprintMission4_/src/services/commentService.js

```diff
@@ -0,0 +1,60 @@
+import prisma from '../lib/prisma.js';
+
+async function registerProductComment(productId, authorId, content){
+  try{
+    const newComment = await prisma.comment.create({
+      data: {
+        content,
+        authorId,
+        productId
+      }
+    })
+    return newComment;
+  } catch(error){
+    throw error;
+  }
+}
+
+async function registerArticleComment(authorId, articleId, content){
+  try{
+    const newComment = await prisma.comment.create({
+      data: {
+        content,
+        articleId,
+        authorId,
+      }
+    })
+    return newComment;
+  } catch(error){
+    throw error;
+  }
+}
+
+async function updateComment(commmentId, content){
+  try{
+    const patchedComment = await prisma.comment.update({
+      where: { id: commmentId},
+      data: { content },
+    })
+    return patchedComment
+  } catch(error){
+    throw error;
+  }
+}
+
+async function deleteComment(commentId){
+  try{
+    await prisma.comment.delete({ where: {commentId}});
```

Comment:
Comment 모델에는, id라는 이름으로 commentId필드가 선언되어있기 때문에, 아래처럼 변경해주어야 동작합니다.

```suggestion
    await prisma.comment.delete({ where: { id: commentId }});
```